ui/src/data-products/kilometer-lengths/kilometer-lengths-container-with-provider.tsx

```diff
@@ -0,0 +1,20 @@
+import { connect, Provider } from 'react-redux';
```

Comment:
Tästä eteenpäin useamman tiedoston verran täysin uutta 🥩:ä. Ei pitäisi tosin sisältää mitään kauhean kummallista, sillä ne on tehty hyvin pitkälti elementtilistan ja pystygeometrioiden pohjalta.